src/pybind/mgr/orchestrator/module.py

```diff
@@ -706,15 +706,12 @@ def _apply_osd(self,
                    format: Format = Format.plain,
                    unmanaged: Optional[bool] = None,
                    dry_run: bool = False,
-                   no_overwrite: bool = False,
-                   inbuf: Optional[str] = None) -> HandleCommandResult:
```

Comment:
There is still a reference to -i in the restrictions mutexes section of the usage description.

```
Restrictions:
  Mutexes:
  * -i, --all-available-devices
  * -i, --unmanaged (this would overwrite the osdspec loaded from a file)
```

Should we also add something to that usage statement (and maybe docs as well) specifying that "orch apply osd" requires using the --all-available-devices flags and any other osd deployment that isn't intended for all other devices needs to go through "ceph orch apply -i"?